src/main/java/seedu/weme/logic/prompter/commandprompter/memecommandprompter/MemeLikeCommandPrompter.java

```diff
@@ -29,7 +29,9 @@ public CommandPrompt prompt(Model model, String userInput) throws PromptExceptio
         }
 
         try {
-            Integer.parseInt(arguments.trim());
+            if (!arguments.trim().matches("\\d+") || Integer.parseInt(arguments.trim()) == 0) {
```

Comment:
might want a better check for this, if -1? if > size of list?

Reply:
My take is that they can key in any positive integer. If they try to execute `like [Invalid INDEX]` then it cannot be executed.